src/ga.js

```diff
@@ -179,7 +187,7 @@ function sendBidRequestToGa(bid) {
   if (bid && bid.bidderCode) {
     _analyticsQueue.push(function () {
       _eventCount++;
-      window[_gaGlobal]('send', 'event', _category, 'Requests', bid.bidderCode, 1, _disibleInteraction);
+      window[_gaGlobal]( trackerSend, 'event', _category, 'Requests', bid.bidderCode, 1, _disibleInteraction);
```

Comment:
`trackerSend` is defined on the `options` object at line [42](https://github.com/prebid/Prebid.js/pull/292/files#diff-b27ea87754623409792c6f0a77373f5cR42), it's not accessible any other way at the moment. The `window[_gaGlobal]` function is called when the `_analyticsQueue` is iterated in the `checkAnalytics` function, so scope has changed a few times over by this point. I've attached two screenshots that illustrate this.

<img width="768" alt="screen shot 2016-04-06 at 10 57 12 pm" src="https://cloud.githubusercontent.com/assets/4116940/14342472/b232c2c6-fc4e-11e5-8f42-3c4078778875.png">
<img width="761" alt="screen shot 2016-04-06 at 11 03 53 pm" src="https://cloud.githubusercontent.com/assets/4116940/14342473/b23450dc-fc4e-11e5-8295-6a5adb15b251.png">

This could be made to work by declaring `trackerSend` as a local variable in the `ga` module's scope, as you do with `_trackerPrefix`. Then it will be accessible to `enableAnalytics` and all the `sendToGa` functions.